### PR TITLE
Tweak texture positions for quick craft

### PR DESCRIPTION
**Pull request by Emirlol**
image
Or with colored pixels for better margin visibility:
image
4 from the right and top as the inventory slots generally have a 4 pixel margin around before the highlights/shadows, then there's 6 pixels from the crafting result and 2 from the button below to group the button together with the quickcraft slots while the button itself has 4 pixels from below

### CODE REVIEW

**Comment by viciscat** [comment]
i liked it popping out :(

**Comment by Julienraptor02** [comment]
> i liked it popping out :(

make it an option :)

**Comment by Julienraptor01** [comment]
> i liked it popping out :(

make it an option
also mojang #whenDataDrivenUIFromBedrockInJava they should port the good stuff smh instead of faking the UIs in the vids

File: src/main/java/de/hysky/skyblocker/skyblock/item/SkyblockCraftingTableScreen.java [hunk 1]
```diff
@@ -194,7 +190,7 @@ public ItemStack quickMove(PlayerEntity player, int slot) {
         }
 
         @Override
-        public boolean canUse(PlayerEntity player) {
+        public boolean canUse(PlayerEntity player)  {
```
Comment (AzureAaron) [inline src/main/java/de/hysky/skyblocker/skyblock/item/SkyblockCraftingTableScreen.java:193]: was this extra space intentional? lol

**Review by AzureAaron — DISMISSED** [review]
looks good

**Review by AzureAaron — APPROVED** [review]

**Pull request merged**